src/ipfs.io/legal/index.html

```diff
@@ -0,0 +1,18 @@
+---
+baseurl: ..
+site: ipfs
+template: tmpl/layouts/ipfs.html
+tagline:
+title: Legal
+---
+{% include "tmpl/partials/pagenav.html" %}
+
+<div class="col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2" style="font-family: Oxygen;">
+{% markdown %}
+
+## Legal
+
+- [Report Copyright Infringement on the IPFS Gateway Service](https://docs.google.com/forms/d/1fXchtQEQ6WDjedSk46ZZiye8vl4UxvyRZiltSOv2huc/viewform)
```

Comment:
Should have another bullet: 

- For all other inquiries, contact: `legal@ipn.io`